components/notification-sender-config/org.wso2.carbon.identity.notification.sender.tenant.config/src/test/java/org/wso2/carbon/identity/notification/sender/tenant/config/handlers/DefaultChannelConfigurationHandlerTest.java

```diff
@@ -130,6 +132,12 @@ public void setUp() {
                 .setResourceManager(resourceManager);
     }
 
+    @Test
+    public void testGetName() {
+
+        Assert.assertEquals(defaultChannelConfigurationHandler.getName(), "default");
```

Comment:
can use the constant here as well.

Reply:
resolved in https://github.com/wso2-extensions/identity-event-handler-notification/pull/184/commits/2f31cb98407af780d8f9b3407a8a957cf4b3ba56